### PR TITLE
Load spec files from resource path if it's provided

### PR DESCRIPTION
**Pull request by mmurdoch**
This lets you update specs without having to re-build everything and
leads to a much faster development cycle. Fixes issue #67.

### CODE REVIEW

File: spec/spec-bootstrap.coffee [hunk 1]
```diff
@@ -35,7 +38,10 @@ else
   require '../vendor/jasmine/lib/jasmine-2.1.3/jasmine-html'
   require '../vendor/jasmine/lib/jasmine-2.1.3/boot'
 
-  for specFilePath in fs.listTreeSync('spec/') when /-spec\.(coffee|js)$/.test specFilePath
+  specRootPath = 'spec/'
+  if global.loadSettings.resourcePath
```
Comment (paulcbetts) [inline spec/spec-bootstrap.coffee]: `resourcePath` should always be set, if it's not you've got Big Problems

**Comment by stefanbuck** [comment]
Is this not already covered by the `--spec-directory` argument?

```
./EightOhEight.app/Contents/MacOS/EightOhEight --help

Options:
  -d, --dev             Run in development mode.                                                           
  -h, --help            Print this usage message.                                                          
  -l, --log-file        Log all output to file.                                                            
  -r, --resource-path   Set the path to the App source directory and enable dev-mode.                      
  -s, --spec-directory  Set the directory from which to run package specs (default: Atom's spec directory).
  -t, --test            Run the specified specs and exit with error code on failures.                      
  -v, --version         Print the version.   
```

**Comment by paulcbetts** [comment]
We should :fire: `--spec-directory` altogether and always run from resource-path. @mmurdoch, can you do that in this PR too?

**Comment by hongrich** [comment]
@mmurdoch actually made the PR from my branch. I just added a commit that :fire: `--spec-directory`. Take another look.

**Comment by paulcbetts** [comment]
Thanks @mmurdoch

**Comment by hongrich** [comment]
Thanks for merging this in, @paulcbetts 👍

**Pull request merged**